services/web-app/components/page-description/page-description.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { Column, Grid } from '@carbon/react'
+import clsx from 'clsx'
+
+import styles from './page-description.module.scss'
+
+const PageDescription = ({ children, className, ...rest }) => (
+  <Grid>
+    <Column sm={4} md={6} lg={8} className={clsx(styles.pageDescription, className)} {...rest}>
+      {children}
+    </Column>
+  </Grid>
+)
+
+export default PageDescription
```

Comment:
It will render a p tag by default
![Screen Shot 2022-04-13 at 3 44 25 PM](https://user-images.githubusercontent.com/2753488/163267284-dc5c9429-654f-4a46-9507-5c639ab84214.png)